microsoft-edge/devtools-guide-chromium/storage/indexeddb.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: View and change IndexedDB data
-description: How to view and change IndexedDB data with the Application panel and Snippets.
+description: How to view and change IndexedDB data with the Application tool and snippets.
```

Comment:
```suggestion
description: How to view and change IndexedDB data by using the Application tool and snippets.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with

alt (tech review this):
```
description: How to view and change IndexedDB data by using the Application tool or snippets.
```